### DISCUSSION TITLE
Wyze v3 on UCTRONICS adapter connection issues

### DISCUSSION THREAD

**Discussion by tismofied** [themactep/thingino-firmware#335] [General]
I have a couple of wyze v3 with thingino. I am using the UCTRONICs poe adapter for power and data. 
They are all connected to a poe switch with a budget of 64w. There is no usb OTG cables in between. If they are all started at the same time, they work for roughly 4 to 5 hours then they start losing connection.
I can’t figure out why they would work for hours then start dropping off network.
The poe adapter still shows active on the router
How do I troubleshoot this issue please?
I would like to know what log to read right before the camera lost connection if possible.

**Comment by themactep** [comment]
there are three major loggers in thingino accessible via `dmesg`, `logread`, `logcat`

**Comment by gtxaspec** [comment]
Make sure your POE switch actually puts out 64w in the budget... I've had some generic switches that can not sustain the claimed power output for long.

**Comment by tismofied** [comment]
Thank you both. The switch I have is [tp-link LS108GP](https://a.co/d/cAwg4RS)
It's a decent unmanaged poe switch. 
I am leaning towards giving up on the poe option altogether and just using wifi. Picture Quality gonna be reduced but it's better than nothing. 
With the adapters, there is too many points of failure and since I am planing on setting these v3s through the attic, I rather have something that just works.
Thanks guys for your suggestions.

**Comment by gtxaspec** [comment]
moving this to discussions for visibility

**Comment by calimansi** [comment]
I'm still having issues too.  Power up my wyze v3 with uctronics and it will stay connected for 5 to 6 hours, then disconnect with a "cdc_ether disconnect" message in dmesg.  The wifi connection still works fine though.  No rebooting.  Wifi is more stable than PoE using the uctronics.  Maybe I'm missing something?  I did use the RtPGtoolUI tool to "fix" the adapter as well.

**Comment by MomosX** [comment]
I just got a hold of some uctronics adapters.   And having the same issues.  They work for a few hours and then drop, but the adaptor still shows in the router table.  Ping is not working anymore , webui also dead.  Testing now 4 uctronics, 2 connected to a POE switch, 2 connected to another POE+ switch.   There is no pattern that i can see.  
POE switches are ok as i have a dahua camera connected to each one and those never drop out.
Did you find a solution for this please ?
